base/applications/msconfig/autochkpage.c

```diff
@@ -0,0 +1,247 @@
+/*
+ * PROJECT:     ReactOS Applications
+ * LICENSE:     LGPL - See COPYING in the top level directory
+ * PURPOSE:     Auto Check (autochk) page handler
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "precomp.h"
+#include "resource.h"
+#include "autochkpage.h"
+
+#include <tchar.h>
+#include <windows.h>
```

Comment:
You shouldn't need windows.h after including precomp.h. If you get an error after you remove this, perhaps you need an additional component header such as winnls.h or winuser.h?